application/templates/static_site/topic.html

```diff
@@ -10,6 +10,15 @@
 {% block socialTitle %}{{ topic.title }}{% endblock %}
 {% block googleAnalytics %}ga('set','contentGroup1','Topic');{% endblock %}
 
+{% block socialDescription %}{% if topic.meta_description %}{{ topic.meta_description }}{% endif %}{% endblock %}
+{% block socialMetadata %}
```

Comment:
I did this slightly differently in https://github.com/racedisparityaudit/ethnicity-facts-and-figures-publisher/pull/1053 (which hasn't been merged yet). Open to ideas on how best to implement this though.

Reply:
Yeah, I decided to do it slightly differently because it feels like having meta tags with no content is weird? We probably don't want to have an element at all if there's nothing to put in it.

But maybe I should have just been consistent with what we already have, and fix it up as a separate thing. I'll do that for now.

Reply:
Ok cool. Let's work out the best strategy for the meta stuff separately?